docs/howto/recalculate_datasets.md

```diff
@@ -0,0 +1,19 @@
+# How to recalculate a Dataset
```

Comment:
There are two 'recalculations' in Horreum:

1) Transforming Runs into Datasets
2) Creating DataPoints out of DataSets and subsequent Change Detection analysis

You probably have a point that we should use consistent naming, maybe come up with better terms (use Retransform for the former?). However in the documentation we need to treat these individually.

Reply:
I used the text on the blue button as a basis as you noticed. I'll go back and edit the button to use your suggestion of "Re-transform" along with the text here. Keeping it consistent.

Reply:
Also will remove any overlap with 2.